process/cla_faq.md

```diff
@@ -0,0 +1,21 @@
+# Contributor License Agreement (CLA) FAQ
+
+Many of the projects hosted by ASWF leverage a Contributor License Agreement for contributors to the given project. These projects use [EasyCLA](https://lfcla.com/) for automating the process of signing and managing CLAs for the project. 
+
+This document captures some commonly asked questions by organizations on CLAs and using the EasyCLA system.
+
+Q: Why does the project require me to sign a CLA?
+
+A: It depends from project to project, but generally the project is looking to ensure that each indivudual is properly authorized to contribute to the project ( most specifically by thier employer ) and that any additional terms that a contributor should be bound to.
```

Comment:
typo: "indivudual" => "individual"

I would probably change this to something like: "A: It depends from project to project. Typically, projects that use CLAs do so because the project community wants to ensure that there is a signed agreement explicitly stating the legal terms governing the contribution to the project. CLAs can also be used by organizations in connection with specifying which of their employees are permitted to contribute code on their behalf, under that CLA."